src/components/Pivot/Pivot.tsx

```diff
@@ -102,7 +102,13 @@ export class Pivot extends React.Component<IPivotProps, IPivotState> {
    */
   private _renderLink(link: IPivotItemProps) {
     const itemKey = link.itemKey;
```

Comment:
can be reduced into a destructuring statement
const { itemKey, itemCount } = link;